src/ChoiceList.re

```diff
@@ -32,41 +38,87 @@ let make = (_children) => {
       ReasonReact.Update({...state, label: Some(choice)})
       } : ReasonReact.Update({...state, label: None})
     | UpdateInput(s) => ReasonReact.Update({...state, input: s})
+    | Weissbier => ReasonReact.Update({...state, labels: ["Weissbier", ...state.labels]})
```

Comment:
Weißbier

Reply:
ja dang das zeigts mir immer falsch an, hab mal gekuckt, hab nur gefunden, dass man das in irgendwelchen .jsp files fixen kann mit nem Kommando

Reply:
Ü, ü     \u00dc, \u00fc
Ä, ä     \u00c4, \u00e4
Ö, ö     \u00d6, \u00f6
ß        \u00df

Might work

Reply:
didnt :/ already looked this up